en/documentation/success-stories/index.md

```diff
@@ -46,9 +46,9 @@ you’ll find a small sample of real world usage of Ruby.
 #### Web Applications
 
 * [Basecamp][9], a web-based project management application developed by
-  [37signals][10], is programmed entirely in Ruby.
+  [Basecamp][10] (previously known as [37signals][11]), is programmed entirely in Ruby.
```

Comment:
I believe their former name is not important. How about:

> Basecamp, a web-based project management application, is programmed entirely in Ruby.

Reply:
@sorah That works!

Reply:
@sorah Adjusted. Thanks!